config/initializers/postgres_bigints.rb

```diff
@@ -0,0 +1,8 @@
+# Rails has a protection to not lookup values by a large number.
+# A lookup/comparison with a large number (bigger than bigint)
+# needs to cast the db column to a double/numeric.
+# and that casting skips the index and forces a table scan
+#
+# https://discuss.rubyonrails.org/t/cve-2022-44566-possible-denial-of-service-vulnerability-in-activerecords-postgresql-adapter/82119
+#
+ActiveRecord::Base.raise_int_wider_than_64bit = false
```

Comment:
@kbrock Did this exist before rails 6.1.7.1?  Do we need to bump the minimum version?

Reply:
don't think so.

I'll try it on earlier versions and go from there

Reply:
Yes, we need to bump minimum as this is brand new to 6.1.7.1.